setup.py

```diff
@@ -26,8 +26,8 @@
     license="AGPLv3",
     author="Overhang.IO",
     author_email="contact@overhang.io",
-    maintainer="Overhang.IO",
-    maintainer_email="regis@overhang.io",
+    maintainer="Edly.IO",
```

Comment:
It should really be "Edly", and not "Edly.IO", because that's how we refer to Edly everywhere -- as opposed to Overhang.IO, where "IO" is part of the name.